packages/Graph/Actions/BaseMsGraphCustomAction.cs

```diff
@@ -22,6 +22,11 @@ namespace Microsoft.Bot.Component.Graph.Actions
     /// </summary>
     public abstract class BaseMsGraphCustomAction : Dialog
     {
+        /// <summary>
+        /// Name of the id field that Graph uses to identify objects in the system.
+        /// </summary>
+        public const string DefaultIdField = "id";
```

Comment:
Rename to IdPropertyName. Alternatively, consider adding a PropertyNames static class and add `public const string Id = "id";` there, then reference that here.

Reply:
Do we really need a class for just one property right now?

Reply:
This is one of this thing where we can break it out into a different class if we see a need for it. And there isn't any adverse side effect if we don't do it.